spec/facter/resolvers/networking_spec.rb

```diff
@@ -5,6 +5,10 @@
 
   let(:log_spy) { instance_spy(Facter::Log) }
 
+  before do
+    pending 'contains failing tests to reproduce a bug'
```

Comment:
This will cause all passing tests to fail, because they're "no longer" pending. See https://rspec.info/features/3-12/rspec-core/pending-and-skipped-examples/pending-examples/ 

You'll want to move the call to `pending` to the specific tests that aren't working yet (instead of callling `raise` in the test) like:

```ruby
 it 'returns dhcp server ip as nil' do
  pending('not working yet')
  expect(networking.resolve(:dhcp)).to be(nil)
end
```

Reply:
Hey @joshcooper! I have put the `pending` call on purpose in the `before` clause because all tests are actually failing with the updated `ifconfig` fixture to demonstrate that it really breaks Facter.
To not put the `pending` call in each and every test case I have moved it to `before`, but for that reason I also had to temporarily add the `raise`s in these two tests to make `rake spec` succeed.
In the following commit I revert the `pending` and `raise`s. Do you want me to move the `pending` call to every test case instead?